tests/integration/ha_tests/helpers.py

```diff
@@ -612,10 +611,7 @@ async def update_service_logging(ops_test: OpsTest, unit, logging: bool):
             continue
         line = line.replace("\n", "")
 
-        if logging:
-            if LOGGING_OPTIONS not in line:
-                mongodb_service[index] = line + " " + LOGGING_OPTIONS + "\n"
-        else:
+        if not logging:
```

Comment:
@dmitry-ratushnyy - I think it is enough to remove the function `update_service_logging ` all together.

IIRC - this function was made so that we could temporarily turn on logs for the tests. Now that we support logs, I think we no longer need to turn on / off logs for the tests.

Mind removing the usage of this function from the tests suite and confirming?

Reply:
Done